src/test/java/com/synopsys/integration/stepworkflow/AbstractConsumingSubStepTest.java

```diff
@@ -0,0 +1,87 @@
+package com.synopsys.integration.stepworkflow;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.stream.Stream;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+public class AbstractConsumingSubStepTest {
+
+    private final String verificationData = "Verification output";
+    private final RuntimeException verificationException = new RuntimeException("Verification runtime exception");
+    private final SubStepResponse<Object> subStepResponse_successOutput = new SubStepResponse<>(true, verificationData, verificationException);
```

Comment:
ConsumingSubSteps should not return data. While testing that it does is technically a valid code path, that is not the intended use of the class and testing it conveys an incorrect API to users of this class. I should consider preventing this in the API of AbstractConsumingSubStep.

Reply:
@rottebds So with the static constructors, I would not be able to test having a SubStepResponse who's subStepSucceeded == true AND has an exception. Of course when subStepSucceeded == true it shouldn't have an exception, but this is allowed with the constructor. If using the static constructors is the intended usage AND the true+exception I mentioned above isn't needed, I will update this asap.

Reply:
@DMaxfield-BDS I retract my previous statement-- the tests as they are now are the best way to test the class. If the class gets updated in the future to make constructing a SubStepResponse "improperly" difficult, then the tests should be updated to reflect that. However, until then these tests are good as-is.